app/Platform/Actions/ComputeGameSortTitleAction.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Actions;
+
+use Transliterator;
+
+class ComputeGameSortTitleAction
+{
+    public function execute(string $gameTitle): string
+    {
+        $sortTitle = $this->padNumbers($gameTitle);
+        $sortTitle = $this->replaceRomanNumerals($sortTitle);
+        $sortTitle = $this->removeArticles($sortTitle);
+        $sortTitle = mb_strtolower($sortTitle);
+        $sortTitle = $this->normalizeAccents($sortTitle);
+        $sortTitle = $this->fixTagTildes($sortTitle);
```

Comment:
Do we also want to strip punctuation?

I'm having trouble coming up with a concrete example of why it would matter. The only thing I can think of is whitespace "game 00001: initial release" vs. "game 00001 - spinoff". Space would sort before colon.

If something does come up, someone could always manually change the sort_titles to fix things.

Reply:
Might as well, it's easy enough to do and probably no harm in doing it. I've updated latest to strip punctuation, only preserving hyphens (-) and tildes (~).

Coverage is thorough enough this didn't require any new tests; the change impacted roughly 13 test cases: https://github.com/RetroAchievements/RAWeb/pull/2690/commits/d05e6337315226add898050f215477c44ca2a3e3